Makefile.e2e

```diff
@@ -59,17 +59,34 @@ test-e2e-gke-nobuild:
 .PHONY: test-e2e-gke
 test-e2e-gke: config-sync-manifest test-e2e-gke-nobuild
 
+KIND_IMAGE := "kind-image"
+
+.PHONY: build-kind-image
+build-e2e-kind: config-sync-manifest-local build-gke-e2e
```

Comment:
Why does it need `build-gke-e2e`?

nit: Can we use the same style, either `build-gke-e2e|build-kind-e2e`, or `build-e2e-gke|build-e2e-kind`?

Reply:
> Why does it need build-gke-e2e?

Got it answered. The kind dockerfile depends on the e2e image, which is built by build-gke-e3e.

Reply:
done